pdr_backend/analytics/predictoor_dashboard/dash_components/callbacks.py

```diff
@@ -0,0 +1,177 @@
+from dash import Input, Output, State
+import dash
+from pdr_backend.analytics.predictoor_dashboard.dash_components.util import (
+    get_feeds_data_from_db,
+    get_predictoors_data_from_db,
+    get_payouts_from_db,
+    filter_objects_by_field,
+)
+from pdr_backend.analytics.predictoor_dashboard.dash_components.view_elements import (
+    get_graph,
+)
+from pdr_backend.analytics.predictoor_dashboard.dash_components.plots import (
+    get_figures,
+)
+
+
+# pylint: disable=too-many-statements
+def get_callbacks(app):
+    @app.callback(
+        Output("feeds-data", "data"),
+        Output("predictoors-data", "data"),
+        Output("error-message", "children"),
+        Input("data-folder", "data"),
+    )
+    def get_input_data_from_db(files_dir):
+        feeds_data = get_feeds_data_from_db(files_dir)
+        predictoors_data = get_predictoors_data_from_db(files_dir)
+        if isinstance(feeds_data, Exception):
```

Comment:
I would not return an Exception, but rather catch a thrown exception. You can use the exception string here, at a later time, and it also becomes apparent what the H3 contains.